src/incflo_field_repo.cpp

```diff
@@ -14,9 +14,8 @@ void incflo::declare_fields()
         "ICNS-" + scheme, m_time, m_repo, m_probtype);
     m_scalar_eqns.emplace_back(amr_wind::pde::PDEBase::create(
         "Temperature-" + scheme, m_time, m_repo, m_probtype));
-
-    const int nstates = 2;
-    m_repo.declare_cc_field("conv_density", 1, 0, nstates);
+    m_scalar_eqns.emplace_back(amr_wind::pde::PDEBase::create(
```

Comment:
This only needs to happen if `m_constant_density == false` correct?

Reply:
I tried that and it didn't work... I think because it creates density_conv_term, should I put if statements to use conv_density when m_constant_density = false, and density_conv_term otherwise?

Reply:
Ah I see. I think this will go away when we move more stuff into equation systems.